scripts/globals/gear_sets.lua

```diff
@@ -256,6 +258,27 @@ function HandleHipsterSet(player, gearset, matches)
         -- printf("we have a special snowflake | gearset: %u | mod %u %u", gearset.id, MOD_FASTCAST, modValue);
         player:addGearSetMod(gearset.id, MOD_FASTCAST, modValue);
         return;
+    elseif (gearset.id == 113) then
+        local modValue = 0;
+        --  Phorcys Armor Set: Set Increases STR, VIT, INT, MND. Bonus is +2 Stats for 2 pieces, +5 for 3 pieces, +10 for 4 pieces, and +15 for 5 pieces
+        if (matches > 1 and matches < 3) then
```

Comment:
Think it would also be possible to table the mods themselves directly, since all 4 stats adjust the same? Sorry, still new to tables, so unsure how to get it to properly work.
Sadly, I tested, and it was not working yesterday (With the if statements), so not sure why this is broken in the first place...